<?xml version="1.0"?>
<launch>

  <include file="$(find vizzy_calibration)/launch/ar_track_alvar.launch">
     <arg name="tf_prefix" value="first" />
     <arg name="node_name" value="first_ar_tracker_alvar" />
     <arg name="cam_image_topic" value="/camera/rgb/image_rect_color" />
     <arg name="cam_info_topic" value="/camera/rgb/camera_info" />		
  </include>
  <include file="$(find vizzy_calibration)/launch/ar_track_alvar.launch">
     <arg name="tf_prefix" value="second" />
     <arg name="node_name" value="second_ar_tracker_alvar" />
     <arg name="cam_image_topic" value="/vizzy/l_camera/image_rect_color" />
     <arg name="cam_info_topic" value="/vizzy/l_camera/camera_info" />	
  </include>


  <node name="vizzy_calibration" pkg="vizzy_calibration" type="two_camera_extrinsic_calibration" output="screen">
     
    <!-- CONFIG PARAMETERS -->
    <param name="number_of_points" value="60" />  <!-- Minimum number of samples for the transformation calculation -->
    <param name="number_of_markers" value="1" />  <!-- Minimum number of samples for the transformation calculation -->
    <param name="first_camera_frame" value="camera_link" />  <!-- camera frame -->
    <param name="second_camera_frame" value="waist_link" />  <!-- end effector frame, where the AR marker should be placed -->
    <param name="marker_link_prefix" value="ar_marker_" /> <!-- marker id --> 

  </node>
</launch>
